fhir-operation-bulkdata/src/main/java/com/ibm/watson/health/fhir/operation/bullkdata/client/BulkDataClient.java

```diff
@@ -188,7 +188,8 @@ public String submit(MediaType outputFormat, Instant since, List<String> types,
         // From the response
         String jobId = Integer.toString(response.getInstanceId());
 
-        return "/wh-fhir-dev/api/v4/$export-status?job=" + jobId;
+        String hostname = properties.get(BulkDataConfigUtil.SERVER_HOSTNAME);
+        return "https://" + hostname + "/wh-fhir-dev/api/v4/$export-status?job=" + jobId;
```

Comment:
Cool - I think we could <config> wh-fhir-dev

Reply:
because /wh-fhir-dev/ could also be changed, e.g, to make it also works in local test env, I think we should store both host name and  "/wh-fhir-dev/api/v4/$export-status?job=" together and use the following pattern:

return "https://" + exportUrl + jobId;

I can make the change in my next PR

Reply:
@albertwang-ibm I raised PR80 for this